packages/strapi-plugin-documentation/services/Documentation.js

```diff
@@ -20,7 +20,12 @@ const defaultSettingsKeys = Object.keys(defaultSettings);
 
 module.exports = {
   areObjectsEquals: (obj1, obj2) => {
-    return JSON.stringify(obj1) === JSON.stringify(obj2);
+    // stringify to remove nested empty objects
+    return _.isEqual(JSON.parse(JSON.stringify(obj1)), JSON.parse(JSON.stringify(obj2)))
```

Comment:
Do you really need the JSON.parse of JSON.stringify here ?

Reply:
Hi, checking on this again, JSON.stringify works on its own apparently, but it isn't consistent, it still will fail if there is more than a single tag and isn't ordered in the same way. I had to create a custom comparator to check for arrays in the object
![image](https://user-images.githubusercontent.com/20282317/92008773-27569480-ed7a-11ea-80ed-98dc2222922a.png) 
![image](https://user-images.githubusercontent.com/20282317/92008976-64228b80-ed7a-11ea-8b77-501e10da334a.png)